src/lib.rs

```diff
@@ -337,6 +348,15 @@ impl PosixACL {
         }
         Ok(())
     }
+
+    /// Returns a raw `acl_t` type that can be used directly in FFI calls to the acl library.
+    ///
+    /// The caller *must not*:
+    /// - `acl_free` or otherwise invalidate the returned `acl_t`
+    /// - use this `acl_t` after the PosixACL has been dropped
+    pub fn as_raw(&self) -> acl_sys::acl_t {
```

Comment:
It seems this method should be marked `unsafe`, what do you think?

Just wondering, what use case do you have for this? Maybe it makes sense to implement your needed feature directly in posix-acl, in addition to this.

Reply:
Moved discussion to other PR.